daemons/mrpd/msrp.c

```diff
@@ -3625,8 +3790,18 @@ int msrp_init(int msrp_enable)
 	if (NULL == MSRP_db)
 		goto abort_socket;
 
+	if( max_interesting_stream_ids < 8 ) {
+		/* Allow minimum 8 interesting talker stream ids and 8 interesting listener stream ids */
```

Comment:
Yes, talkerIDs are what the max_interesting applies to. So if you are suggesting I rename the  max_interesting_stream_ids variable, I agree.